docs/users_guide/landice/test_groups/ismip6_forcing.rst

```diff
@@ -0,0 +1,109 @@
+.. _landice_ismip6_forcing:
+
+ismip6_forcing
+============
+
+The ``landice/ismip6_forcing`` test group implements processing of atmospheric
+and ocean forcing data of the Ice Sheet Model Intercomparison for CMIP6
+(ISMIP6) protocol. The ISMIP6 source data can be obtained by directly
+contacting the authors. Reference: https://www.climate-cryosphere.org/wiki/index.php?title=ISMIP6-Projections2300-Antarctica#A2.2_Retrieving_datasets_and_uploading_your_model_output
```

Comment:
I think you should add something here about how a user needs to download the ISMIP6 forcing data in order to use these test cases.  As I understand it, they need to create a base directory that is equivalent to one of the GHub globus endpoints (`GHub-ISMIP6-Forcing` or `ISMIP6 Projections Forcing 2300`).  Then, they need to manually download the files in the same directory structure as on the endpoint.  How do they know what files to download for a given model, scenario, and period end year?  Could you give an example for 2100 and another for 2300?

Reply:
`output_base_path` also needs to be discussed.

Reply:
Don't forget to mention the files in `parameterizations` and `imbie2_basin_numbers_8km.nc`